src/MudBlazor/Components/Hotkey/MudHotkey.razor.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+using Microsoft.JSInterop;
+using MudBlazor.State;
+using MudBlazor.Utilities;
+
+namespace MudBlazor;
+
+#nullable enable
+/// <summary>
+/// Allows registering a hotkey.
+/// </summary>
+public partial class MudHotkey : MudComponentBase, IAsyncDisposable
+{
+    private readonly string _hotkeyId = Guid.NewGuid().ToString();
```

Comment:
I generally prefer using the built in Identifier.
```suggestion
    private readonly string _hotkeyId = Identifier.Create("hotkey");
```

Reply:
What is the advantage of this `Identifier` compared to a guid?

Reply:
It's more readable and easier to identify when something you are looking at in the dom is related or doesn't relate. It's optional and I don't believe yours is used in the dom anywhere. But the primary advantage is readability when you come across it.

Reply:
Not much, mostly it's shorter than guids and can have prefix to help find the id faster for automation tests.

Reply:
Ah okay